.github/workflows/autofix.yml

```
@@ -23,3 +23,5 @@ jobs:
       - name: Fix lint issues
         run: npm run lint:fix
       - uses: autofix-ci/action@8caa572fd27b0019a65e4c695447089c8d3138b9
+        with:
+          commit-message: 'style: apply lint fixes'
```

I am thinking to add `[bot]` in order to be able to filter out this commits from changelog:

```suggestion
          commit-message: 'style: apply lint fixes [bot]'
```

---

sure. interesting idea... (though could you use the github api which tells you if a user is a bot?)

---

Author is also containing bot `Author: autofix-ci[bot] <114827586+autofix-ci[bot]@users.noreply.github.com>` we might as well check that 👍🏼